src/manipulators/deleteIfHandleOutsideLimits.js

```diff
@@ -0,0 +1,32 @@
+import { removeToolState } from '../stateManagement/toolState.js';
+import {
+  anyHandlesOutsideDisplayedArea,
+  anyHandlesOutsideImage,
+} from './index.js';
+
+/**
+ * Delete the measurement if it has any handle outside the image or displayed
+ * area's boundaries depending on the defined handle moving options.
+ * If both deletion flags are false, it will delete the measurement.
```

Comment:
@igoroctaviano I think it should be as:
```
/**
 * Delete the measurement if it has any handle outside the image or displayed
 * area's boundaries depending on the defined handle moving options.
 * If both deletion flags are false, it will not delete the measurement.
```

Reply:
Done!